tests/OnlineSales.Tests/TestEntities/TestCustomer.cs

```diff
@@ -6,10 +6,18 @@
 
 namespace OnlineSales.Tests.TestEntities;
 
-public class TestCustomer : CustomerCreateDto
+public class TestCustomerCreate : CustomerCreateDto
 {
-    public TestCustomer()
+    public TestCustomerCreate()
     {
         Email = "customer@test.net";
     }
+}
+
+public class TestCustomerUpdate : CustomerUpdateDto
```

Comment:
@IgorKostyukov all *Update classes must be removed as they break the concept that update is a partial operation, i.e. I do not want the Update entity to have any default values except these I set inside a particular test. Please also remove *Create in the class names. It should stay TestCustomer, not TestCustomerCreate.